torchaudio/models/wav2vec2/pretrained.py

```diff
@@ -122,6 +122,530 @@ def _get_labels():
+        "aux_num_out": None,
+    },
+    _labels=None,
+)
+WAV2VEC2_BASE.__doc__ = """wav2vec 2.0 model with "Base" configuration.
+
+Trained on 960 hours of *LibriSpeech* [:footcite:`7178964`] dataset. Not fine-tuned.
+
+Originally published by the authors of *wav2vec 2.0* [:footcite:`baevski2020wav2vec`].
+[`Source <https://github.com/pytorch/fairseq/tree/main/examples/wav2vec#pre-trained-models>`__]
+"""
+
+WAV2VEC2_ASR_BASE_10M = Wav2Vec2PretrainedModelBundle(
+    _path='wav2vec2_fairseq_base_ls960_asr_ll10m.pth',
+    _params={
+        'extractor_mode': 'group_norm',
+        'extractor_conv_layer_config': [
+            (512, 10, 5),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 2, 2),
+            (512, 2, 2),
+        ],
+        'extractor_conv_bias': False,
+        'encoder_embed_dim': 768,
+        'encoder_projection_dropout': 0.1,
+        'encoder_pos_conv_kernel': 128,
+        'encoder_pos_conv_groups': 16,
+        'encoder_num_layers': 12,
+        'encoder_num_heads': 12,
+        'encoder_attention_dropout': 0.1,
+        'encoder_ff_interm_features': 3072,
+        'encoder_ff_interm_dropout': 0.0,
+        'encoder_dropout': 0.1,
+        'encoder_layer_norm_first': False,
+        'encoder_layer_drop': 0.05,
+        "aux_num_out": 32,
+    },
+    _labels=_get_labels(),
+)
+WAV2VEC2_ASR_BASE_10M.__doc__ = """Build "base" wav2vec2 model with an extra linear module
+
+Pre-trained on 960 hours of *LibriSpeech* [:footcite:`7178964`] dataset, and
+fine-tuned for ASR on 10 minutes of *Libri-Light* [:footcite:`librilight`] dataset.
+
+Originally published by the authors of *wav2vec 2.0*
+[:footcite:`baevski2020wav2vec`].
+[`Source <https://github.com/pytorch/fairseq/tree/main/examples/wav2vec#pre-trained-models>`__]
+"""
+
+WAV2VEC2_ASR_BASE_100H = Wav2Vec2PretrainedModelBundle(
+    'wav2vec2_fairseq_base_ls960_asr_ls100.pth',
+    {
+        'extractor_mode': 'group_norm',
+        'extractor_conv_layer_config': [
+            (512, 10, 5),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 2, 2),
+            (512, 2, 2),
+        ],
+        'extractor_conv_bias': False,
+        'encoder_embed_dim': 768,
+        'encoder_projection_dropout': 0.1,
+        'encoder_pos_conv_kernel': 128,
+        'encoder_pos_conv_groups': 16,
+        'encoder_num_layers': 12,
+        'encoder_num_heads': 12,
+        'encoder_attention_dropout': 0.1,
+        'encoder_ff_interm_features': 3072,
+        'encoder_ff_interm_dropout': 0.0,
+        'encoder_dropout': 0.1,
+        'encoder_layer_norm_first': False,
+        'encoder_layer_drop': 0.05,
+        "aux_num_out": 32,
+    },
+    _labels=_get_labels(),
+)
+
+WAV2VEC2_ASR_BASE_100H.__doc__ = """Build "base" wav2vec2 model with an extra linear module
+
+Pre-trained and fine-tuned for ASR on 960 hours of
```

Comment:
I think this is switched with the `WAV2VEC2_ASR_BASE_960H` [doc](https://github.com/pytorch/audio/pull/1827/files#diff-f8152dc5c8f7b34fdd59cc08f75744d400f06235470b88f3b6076d3523db97ebR275) below

Reply:
Good catch! Thank you!